server/src/test/java/io/crate/execution/engine/collect/GroupByOptimizedIteratorTest.java

```diff
@@ -73,9 +74,11 @@
     private String columnName;
     private InputCollectExpression inExpr;
     private List<AggregationContext> aggregationContexts;
+    private NodeContext nodeCtx;
 
     @Before
     public void prepare() throws Exception {
+        nodeCtx = createNodeContext();
```

Comment:
Could use a local variable, the class attribute is not used as far as I can tell